handlers/main.yml

```diff
@@ -78,3 +78,6 @@
 
 - name: reset audit pkg ugids
   command: rpm --setugids audit
+
+- name: remount /tmp
```

Comment:
Does the mount module not work here? http://docs.ansible.com/ansible/mount_module.html